app/models/auction.rb

```diff
@@ -1,2 +1,8 @@
 class Auction < ApplicationRecord
+  belongs_to :user
+  has_many :bids
+
+  def bid
```

Comment:
what about, `last_bid`, `current_bid` or something that makes more clear which of all bids it is?

Reply:
Is there a way to alias it under another name in the serializer? Whatever I name it here, the serializer will add a key to the returned object with that same name.

Reply:
I guess only if you list it under `attributes`, right?

Whatever you name this method, is what you'd use in `has_one`.